src/core/manager.h

```diff
@@ -448,6 +448,8 @@ struct Manager {
         VarlinkServer *varlink_server;
         /* Only systemd-oomd should be using this to subscribe to changes in ManagedOOM settings */
         Varlink *managed_oom_varlink_request;
+        /* Used by user managers to send ManagedOOM settings changes to systemd-oomd. */
```

Comment:
conversely menion here that this is a varlink connection where we are the client, and we connected to oomd, and that we use this only if we run as user manager